ci/check-documentation.py

```diff
@@ -114,8 +114,25 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
-        # check for existence of links to README.md files of models and demos
+        # check <omz_dir> link validity
+
+        for link in sorted([link for link in doc_page.omz_references() if link.startswith('<omz_dir>')]):
+            file_path = Path(link.replace('<omz_dir>', str(OMZ_ROOT)))
+
+            try:
+                file_relative_path = file_path.relative_to(OMZ_ROOT)
+            except ValueError:
+                complain(f'{md_path_rel}: invalid OMZ reference {file_path!r}')
+                continue
+
+            if str(file_relative_path) == md_path_rel: # self-link
+                continue
```

Comment:
What's the purpose of this check? It's true that a self-link should always be valid and thus doesn't _need_ to be checked, but the cost for checking it anyway is negligible, so there's no need to special-case it. (Also, I'm not sure why we would even have any `<omz_dir>` references that point to the document they're in.)